extensions/tls-registry/runtime/src/main/java/io/quarkus/tls/runtime/config/TlsConfigUtils.java

```diff
@@ -78,7 +82,15 @@ public static void configure(TCPSSLOptions options, TlsConfiguration configurati
                 options.addCrlValue(buffer);
             }
             options.setEnabledSecureTransportProtocols(sslOptions.getEnabledSecureTransportProtocols());
-            options.setUseAlpn(sslOptions.isUseAlpn());
+            // Try to set ALPN configuration, but handle UnsupportedOperationException
+            // for example, if the underlying implementation does not support it (es. AMQP)
+            try {
+                options.setUseAlpn(sslOptions.isUseAlpn());
+            } catch (UnsupportedOperationException e) {
+                log.warn(
```

Comment:
Can you use warnf instead?

Reply:
Yes, sorry, you're right. I made the change.